datawig/simple_imputer.py

```diff
@@ -148,7 +148,7 @@ def _is_categorical(col: pd.Series,
 
         """
 
-        return col.sample(n=n_samples, replace=len(col) < 100).value_counts(
+        return col.sample(n=n_samples, replace=len(col) < n_samples).value_counts(
```

Comment:
Do we have a test for it?

Reply:
https://github.com/awslabs/datawig/blob/580f7a3d41430b180002e51b16f90e2b26af3be0/test/test_simple_imputer.py#L285